outreachy-round-13.md

```diff
@@ -83,15 +89,60 @@ be skilled in writing in English for a non-academic audience, and to
 be interested in getting feedback that will help them write accurately
 and accessibly about technology.
 
-We'll be asking for you to publicly post a few sections of your
-proposal -- the project summary, list of deliverables, and timeline --
-someplace public on the Web, such as on your blog, or in a public
-document on a platform like
-[EtherPad](https://public.etherpad-mozilla.org/) or
-[Hackpad](https://hackpad.com) or [Dropbox
-Paper](https://www.dropbox.com)). That way, the whole developer
-community -- not just the mentors and administrators -- have a chance
-to give you feedback and help you improve your proposal.
+We would like you to publicly post a few sections of your proposal --
+the project summary, list of deliverables, and timeline -- someplace
+public on the Web, such as on your blog, sometime before October
+17th. That way, the whole developer community -- not just the mentors
+and administrators -- have a chance to give you feedback and help you
+improve your proposal. See ["let's get started publishing drafts of
+proposals" on our mailing
+list](https://groups.google.com/forum/#!topic/zulip-outreachy-round-13/dhs2wX6RaSo)
+for more details.
+
+As the [application process
+guide](https://wiki.gnome.org/Outreachy#Application_Process) says:
+
+"While only one contribution is required to be considered for the
+program, we find that the strongest applicants make multiple
+contributions throughout the application process, including after the
+application deadline."
+
+We are more interested in candidates if we see them submitting good
+contributions to Zulip projects, helping other applicants on GitHub
+and on our mailing lists and on
+[zulip.tabbott.net](https://zulip.tabbott.net/#narrow/stream/Outreachy.202016-2017/subject/Welcome),
+learning from our suggestions, and developing and sharing project
+ideas and project proposals that are plausible and useful.
+
+When you are learning how to contribute in open source,
+you're going to find that people give you links to pages that answer
+your questions. Here's how that usually goes:
+
+1) you ask a question
+2) someone directs you to a document
+3) you go read that document, and try to use it to answer your question
+4) you find you are confused about a new thing
+5) you ask another question
+6) now that you have shown that you have the ability to read, think, and
+learn new things, someone has a longer talk with you to answer your new
+specific question
+7) you and the other person collaborate to improve the document that you
+read in step 3 :-)
```

Comment:
can you add a link to that Akamai post?  I think we should encourage folks to search our documentation and struggle a little bit before we give advice with step 1 being "ask a question".